lib/sequence/binlog.js

```diff
@@ -29,6 +29,7 @@ module.exports = function(options) {
   };
 
   Binlog.prototype['OkPacket'] = function(packet) {
+    // TODO: Remove console statements to make module ready for widespread use
     console.log('Received one OkPacket ...');
```

Comment:
Looks like we don't do anything with `OkPacket`s in ZongJi. See the `switch` at line 21.

Here's the definition in `node-mysql`: https://github.com/felixge/node-mysql/blob/master/lib/protocol/sequences/Sequence.js#L36

I would be very surprised if you could figure out a way to get that console message to actually appear. Maybe @nevill has an idea?